.circleci/config.yml

```diff
@@ -159,7 +159,7 @@ jobs:
             source .circleci/local_publish_helpers.sh
             amplify -v
             cd packages/amplify-console-integration-tests
-            retry yarn run console-integration --maxWorkers=3
```

Comment:
--no-cache is used so that Jest doesn't ignore worker pools (by default it will try to use cache info to determine test ordering, so it won't always run tests in parallel)